docs/bots-guide.md

```diff
@@ -346,6 +346,57 @@ client.update_message(dict(
 * My bot works only on some streams.
     * Subscribe your bot to other streams, as described [here](#how-to-run-a-bot).
 
+## Flask Server
+It is a simple python flask server which can be used to interact with bots through making use of
```

Comment:
I would start with a brief discussion of the problem this solves: running one (or more) bots in production.